docs/release_documentation/planning.adoc

```diff
@@ -0,0 +1,133 @@
+//verwende glaube immer unterschiedliche und falsche zeitformen
+
+= Planung
+
+== Ausgangssituation in Software Engineering II
+
+===  Auftraggeber und Aufgabenstellung
+
+Im Rahmen des Moduls "Software Engineering II" haben wir die Aufgabe erhalten eine Art Kalender für T-Systems MMS zu entwickeln. Unser Ansprechpartner in der Firma war Herr Sven Liebert und in der HTW Dresden waren es Herr Zirkelbach als Coach und Herr Prof. Anke als Dozent.
+
+Die Webanwendung solle den Mitarbeitenden ermöglichen ihre geplanten Projekte in einen Kalenderübersicht einzutragen. Die Darstellung soll ohne konkrete Zeit auskommen und soll Darstellen an was der Mitarbeitende an diesen Tag gearbeitet hat bzw. plant zu arbeiten. Dafür sollen Projekte und Mitarbeitern erstellt werden können. Die Mitarbeiter sollen ebenfalls zu Abteilungen zugeordnet werden können. Jeder Mitarbeiter hat soll Projekte und andere Mitarbeiter erstellen bzw. bearbeiten können. Ebenso darf jeder Mitarbeiter, auch für Kollegen, die Projekte für die jeweiligen Tage ein- bzw- austragen können
```

Comment:
vielleicht lieber statt "ohne konkrete Zeit", sowas wie "die kleinste Einheit zur Planung ist dabei in Tagen" oder "die Zuteilungen werden jeweils auf einen Wochentag getroffen"